### ISSUE TITLE
Results Filter for Symptoms of a Disease (MVP1)

### ISSUE THREAD

**Issue by sstemann** [NCATSTranslator/Feedback#199]
When you have a set of results, add the ability to filter the drug by symptoms its treats

**Comment by sierra-moxon** [comment]
@sstemann - should the milestone be "Fall" for this one?  Or does it need to happen pre-September?

**Comment by sstemann** [comment]
if its in the attribute-server currently - how much effort is it to include for September? @gprice1129 @dnsmith124

**Comment by dnsmith124** [comment]
I don't _think_ we have access to this information from the attribute server, but I may be incorrect. @gprice1129 should be able to confirm/deny

**Comment by sstemann** [comment]
@gprice1129 were you able to confirm/deny?

**Comment by Genomewide** [comment]
We do have the 'indicated for' information.  

I thought this would be a limited set, but it is really not for some drugs and will have to be researched for what the best way to implement it is.  

The indicated for may also need some ontology reasoning or may need to be excluded from ontology reasoning.  For example, is insulin 'indicated for diabetes' or is that too broad and it is actually indicated for either type 1 or type 2. 

If so, then does the UI say that insulin is not indicated for diabetes?

Or if is indicated for 'Diabetes' only, do we show it for type 1 or type 2?

This requires some examples and research prior adding this.

**Comment by sandrine-m** [comment]
Changing this feature request to post-release. @sierra-moxon @mbrush should it be a good item for the predicate call?

**Comment by gprice1129** [comment]
@sstemann I am unable to find an attribute for something like "symptoms" provided by the annotation server. However I do see in the metadata fields of mydisease.info some attributes related to phenotypes. These also do not seem to appear as part of the attribute server message. @newgene can you speak on this?

**Comment by sierra-moxon** [comment]
@newgene @gprice1129 - would you mind assessing whether this request is possible before the end of this phase of Translator?

**Comment by newgene** [comment]
yes, we do have phenotypes associated with diseases available from MyDisease.info API, which is not returned as default from the annotator service.

Here are a list of 10,299 disease records with both MONDO ids and `phenotype_related_to_disease` field values (sourced from HPO):

https://mydisease.info/v1/query?q=_exists_:hpo.phenotype_related_to_disease%20AND%20_exists_:mondo&fields=mondo.mondo,mondo.label,hpo.phenotype_related_to_disease

We can evaluate how useful they are, and decide if we should include them in the annotator response.
